app/src/main/java/com/devshish/internship/presentation/ui/profile/ProfileViewModel.kt

```diff
@@ -1,37 +1,55 @@
 package com.devshish.internship.presentation.ui.profile
 
+import androidx.annotation.StringRes
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
+import com.devshish.internship.R
 import com.devshish.internship.domain.model.User
 import com.devshish.internship.domain.repository.IProfileRepository
 import com.devshish.internship.domain.repository.ITokenRepository
+import com.devshish.internship.presentation.ui.utils.Event
 import kotlinx.coroutines.launch
+import timber.log.Timber
 
 class ProfileViewModel(
-    repository: IProfileRepository,
+    private val repository: IProfileRepository,
     private val tokenRepository: ITokenRepository
 ) : ViewModel() {
 
+    sealed class UIState {
```

Comment:
1. It is not state , so again - wrong naming 
2. your event names should name what View should do - for example, instead of NOEXEPTION you can name NavigateTo*** or something like this.
3. Also try to change Exception to something else, because Exception - is common name and the one who will read can misunderstand this. Error could be ok, or something other